tests/test_registerutils.py

```diff
@@ -3239,6 +3239,12 @@ def test_remove_service(
     mock_logging.info.not_called()
 
 
+@patch('socket.create_connection')
+def test_network_access_error(mock_socket_create_connection):
```

Comment:
I'm not against this explicit test

Just mentioning that that code path is tested in `test_fetch_smt_data_api_no_valid_ip` 
Maybe we want to extend that test to check the log message ?

Reply:
I think I'm going to drop this test from here as you covered it in another PR that you did after I created this one

Reply:
done